apps/builder/app/builder/features/sidebar-left/panels/components/components.tsx

```diff
@@ -50,6 +50,14 @@ export const TabContent = ({ publish, onSetActiveTab }: TabContentProps) => {
             if (category === "hidden") {
               return false;
             }
+
+            if (
```

Comment:
just discussed with Bogdan, we should for now keep it hidden, so this can be removed

Reply:
Then we decided to leave under flag for now